avocado_vt/plugins/vt.py

```diff
@@ -1180,24 +1185,23 @@ def configure(self, parser):
                                                   "If --vt-config is "
                                                   "provided, this will be "
                                                   "ignored. Default: ''"))
-        qemu_bin_path_current = settings.get_value('vt.qemu', 'qemu_bin',
-                                                   default=qemu_bin_path)
+        qemu_bin = settings.get_value('vt.qemu', 'qemu_bin',
```

Comment:
How about  qemu_default_path = qemu_bin  or qemu_bin_path ? for that qemu path in configuration file with high priority  that  default qemu binary in  system PATH